packages/firebase_storage/firebase_storage/ios/Classes/FLTFirebaseStoragePlugin.m

```diff
@@ -61,39 +61,47 @@ typedef NS_ENUM(NSUInteger, FLTFirebaseStorageStringType) {
 };
 
 @interface FLTFirebaseStoragePlugin ()
-@property(nonatomic, retain) FlutterMethodChannel *channel;
+@property(nonatomic, retain) FlutterMethodChannel *storage_method_channel;
+//@property(strong, nonatomic) PigeonStorageFirebaseApp *appleArguments;
+
 @end
 
 @implementation FLTFirebaseStoragePlugin {
   NSMutableDictionary<NSNumber *, FIRStorageObservableTask<FIRStorageTaskManagement> *> *_tasks;
   dispatch_queue_t _callbackQueue;
   bool hasEmulatorBooted;
+  NSObject<FlutterBinaryMessenger> *_binaryMessenger;
+  NSMutableDictionary<NSString *, FlutterEventChannel *> *_eventChannels;
+  NSMutableDictionary<NSString *, NSObject<FlutterStreamHandler> *> *_streamHandlers;
```

Comment:
These need to be cleaned up in `cleanupWithCompletion` method.

Reply:
done